text/0000-partial_types.md

```diff
@@ -0,0 +1,642 @@
+- Feature Name: `partial_types`
+- Start Date: 2023-04-28
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+
+# Summary
+[summary]: #summary
+
+Partial Types proposal is a generalization on "partial borrowing"-like proposals.
+
+This proposal is universal flexible tool to work **safe** with partial parameters, partial arguments, partial references and partial borrowing.
+
+Advantages: maximum type safety, maximum type control guarantee, no ambiguities, flexibility, usability and universality.
+
+
+# Motivation
+[motivation]: #motivation
+
+Safe, Flexible controllable partial parameters for functions and partial consumption (including partial not borrowing) are highly needed.
```

Comment:
Why are they needed? The motivation should give a _concrete_ example showing a problem that is solved with the RFC.

Reply:
Thanks for clarification!